indico/modules/designer/placeholders.py

```diff
@@ -71,6 +75,22 @@ def render(cls, registration):
         return name
 
 
+class EventLogoPlaceholder(DesignerPlaceholder):
+    group = 'event'
+    name = 'event_logo'
+    description = _("Event Logo")
+
+    @classmethod
+    def render(cls, registration):
+        try:
+            buf = io.BytesIO(registration.logo)
+            img = Image.open(buf)
+            return img
+        except Exception as e:
+            logger.error('Cannot get Event image logo: %s', e)
```

Comment:
I don't think this try..except is really useful.

- catching any `Exception` is way too broad
- what kind of failures do you expect here? and even if you catch it, returning `None` will most likely break later

So I would just remove this altogether and if something breaks, then you keep a meaningful exception and traceback, which is better than handling it in a way where you either get a failure later or have a broken result (e.g. a badge with a missing image)

Reply:
Yes, the Exception is not nice... I'll change it for an IOException. I want to catch the missing file.
On the other hand... imagine being a Security Officer trying to print a badge with a long queue in front: better get an Indico error, or a badge even without the correct logo? :)
I'll fix the Exception and return something...

Reply:
But... you cannot have a missing file here. the event logo is stored in the database.

What can happen of course is that the event has no logo at all, because none has been uploaded, or someone deleted it. But this case should *not* be handled here - instead, check `event.has_logo` and if there either use a placeholder image (like in case of broken images on the web) or return None **but make sure it's handled cleanly in the code later (which may be hard, because it checks for isinstance Image to know whether it's an image or text)

Reply:
Ah ok. 
I had in mind the logo was on FS.
OK, I'm going to change it

Reply:
I checked it out, returning an empty string will print the badge without logo and no errors.